src/components/DatePicker/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@bcrs-shared-components/date-picker",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@bcrs-shared-components/enums": "^1.0.10",
+    "@bcrs-shared-components/interfaces": "^1.0.16",
```

Comment:
This component doesn't depend on enums or interfaces right?

Reply:
It does not! Sorry, copypasta error.
Will remove until required.